react/features/transcription/index.js

```diff
@@ -0,0 +1,5 @@
+export * from './actions';
+export * from './components';
+
+import './middleware';
+import './reducer';
```

Comment:
If the recording feature includes the transcription toggle, should transcription components be included in recording?